Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+    <string name="label_steps_count">Numero de pasos</string>
+    <string name="label_vehicle_details">Detalles del vehiculo</string>
+    <string name="label_yes_no_dialog_dont_show">No mostrar otra vez</string>
+    <string name="length_label">Longitud</string>
+    <string name="locator">Historial de vuelos</string>
+    <string name="loiter_ccw">CCW</string>
+    <string name="lst_note_hint">Para añadir mas filas , pulsa enter</string>
+    <string name="menu_action_delete">Eliminar</string>
+    <string name="menu_action_multi_edit">Multi-Edit</string>
+    <string name="menu_action_reverse">Dar la vuelta</string>
+    <string name="menu_drone_setup">Configurar Drone</string>
+    <string name="menu_info_bar">Barra de informacion</string>
+    <string name="menu_map_type">Tipo de mapa</string>
+    <string name="menu_map_type_hybrid">Hibrido</string>
+    <string name="menu_map_type_normal">Normal</string>
+    <string name="menu_map_type_satellite">Vista Satelite</string>
+    <string name="menu_map_type_terrain">Terreni</string>
+    <string name="menu_search_toggle"></string>
+    <string name="metadata_range">Rango:</string>
+    <string name="metadata_units">Unidades:</string>
+    <string name="metadata_value">Valor:</string>
+    <string name="metadata_values">Valores:</string>
+    <string name="meter_unit">m</string>
+    <string name="mission_control_arm">Armar</string>
+    <string name="mission_control_auto">Auto</string>
+    <string name="mission_control_change_speed">Cambiar velocidad</string>
+    <string name="mission_control_condition_yaw_angle">Angulo objetivo</string>
+    <string name="mission_control_condition_yaw_relative">Angulo relativo</string>
+    <string name="mission_control_control">Control</string>
+    <string name="mission_control_disarm">Desarmar</string>
+    <string name="mission_control_dronie">Dronie</string>
+    <string name="mission_control_edit">Editar</string>
+    <string name="mission_control_follow">Seguir</string>
+    <string name="mission_control_home">Casa</string>
+    <string name="mission_control_land">Tierra</string>
+    <string name="mission_control_pause">Pausar</string>
+    <string name="mission_control_takeoff">Despegar</string>
+    <string name="mission_length"></string>
+    <string name="mission_is_too_lengthy">La mision es muy larga</string>
+    <string name="mode_acro">Acro permite al piloto tomar el contorl de la rotacion directamente</string>
+    <string name="mode_althold">Alt Hold controla automáticamente el acelerador y mantiene la altitud. No requiere GPS.</string>
+    <string name="mode_auto">Auto sigue la mision almacenada en el vehiculo</string>
+    <string name="mode_circle">Círcle hace girar el vehículo alrededor de un punto de interés. El vehículo siempre apuntará hacia el centro. Requiere GPS conectado.</string>
+    <string name="mode_drift">Drift combina balanceo y guiñada para producir un vuelo más natural e intuitiva. Requiere bloqueo GPS.</string>
```

Comment:
I'm not sure about **guiñada** in here, what do you think if we describe **yaw** as "movimiento de rotación sobre un eje vertical", its way more complex but I think it explains yaw better, another option would be to just call it **yaw**, what do you think?